velox/exec/tests/OperatorUtilsTest.cpp

```diff
@@ -206,3 +207,56 @@ TEST_F(OperatorUtilsTest, gatherCopy) {
 TEST_F(OperatorUtilsTest, makeOperatorSpillPath) {
   EXPECT_EQ("spill/task_1_100", makeOperatorSpillPath("spill", "task", 1, 100));
 }
+
+TEST_F(OperatorUtilsTest, wrap) {
+  auto rowType = ROW({
+      {"bool_val", BOOLEAN()},
+      {"tiny_val", TINYINT()},
+      {"small_val", SMALLINT()},
+      {"int_val", INTEGER()},
+      {"long_val", BIGINT()},
+      {"ordinal", BIGINT()},
+      {"float_val", REAL()},
+      {"double_val", DOUBLE()},
+      {"string_val", VARCHAR()},
+      {"array_val", ARRAY(VARCHAR())},
+      {"struct_val", ROW({{"s_int", INTEGER()}, {"s_array", ARRAY(REAL())}})},
+      {"map_val",
+       MAP(VARCHAR(),
+           MAP(BIGINT(),
+               ROW({{"s2_int", INTEGER()}, {"s2_string", VARCHAR()}})))},
+  });
+
+  for (int32_t iter = 0; iter < 20; ++iter) {
+    VectorFuzzer fuzzer({}, pool());
+    auto vector = CHECK_NOTNULL(fuzzer.fuzzFlat(rowType));
```

Comment:
CHECK_NOTNULL is not need. If it was needed, we'd use VELOX_CHECK_NOT_NULL.

Reply:
I thought to use it avoid some fb internal warning. Will VELOX_CHECK_NOT_NULL does the same thing? Thanks!

Reply:
What's the warning? I think we can just remove this check as it clutters the code. Why do we need it?

Reply:
I removed the check. If a function could returns a nullptr, some meta internal tool warns to add a null check and suggested to use CHECK_NOTNULL which is actually commonly used in initializer in ctor. I am not sure if those tool can recognize VELOX_CHECK_NOTNULL or not.